dashboard.js

```diff
@@ -0,0 +1,32 @@
+function resizeNumberWidgetTextToFit() {
```

Comment:
this we are doing to handle which case?

Reply:
so that the number fits the box widget?

Reply:
This is the default js that is being referenced in the rendered HTML .
We haven't made any changes to js for now. Later, if any changes are to be done, can be done in this.

Reply:
ohk